noxfile.py

```diff
@@ -48,7 +68,9 @@ def lint(session):
     """
     session.install("flake8", "black")
```

Comment:
Because `black` has tended to drift in its spec over time, we've been pinning it to a specific version (defined as a variable to enforce sharing with the `blacken` session below).